project.clj

```diff
@@ -3,12 +3,37 @@
   :url "https://github.com/fredericksgary/test.chuck"
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
-  :dependencies [[org.clojure/clojure "1.6.0"]
+  :dependencies [[org.clojure/clojure "1.7.0"]
+                 [org.clojure/clojurescript "0.0-3308"]
```

Comment:
`:provided` sounds fine, especially if there's precedent for multi-platform libraries.

I don't care at all about test.check uberjars, I don't think that's a use case for anybody. I was only worried about transitive dependencies, both because it's annoying/noisy and also because there's more possibility for version conflicts e.g. if you're using any of the dependencies that cljs itself uses.